js/src/Map.js

```diff
@@ -176,11 +197,31 @@ var LeafletMapView = utils.LeafletDOMWidgetView.extend({
                 var c = e.target.getCenter();
                 that.model.set('center', [c.lat, c.lng]);
                 that.dirty = false;
-            }
+            };
+            var style_view = Object.keys(that.model.get('default_style').views)[0];
+            that.model.get('default_style').views[style_view].then(function(view) {
+                view.style();
+            });
+            var dragging_style_view = Object.keys(that.model.get('dragging_style').views)[0];
+            that.model.get('dragging_style').views[dragging_style_view].then(function(view) {
+                view.unstyle();
+            });
```

Comment:
I guess you need to unstyle before styling, something like:
```javascript
var dragging_style_view = Object.keys(that.model.get('dragging_style').views)[0];
that.model.get('dragging_style').views[dragging_style_view].then(function(view) {
    // First remove the dragging_style
    view.unstyle();

    // Then apply the default style
    var style_view = Object.keys(that.model.get('default_style').views)[0];
    that.model.get('default_style').views[style_view].then(function(view) {
         view.style();
    });
});
```

And something equivalent on `movestart`: `unstyle` the `default_style` and `style` the `dragging_style`.